spec.go

```diff
@@ -0,0 +1,94 @@
+// Copyright © 2022 Meroxa, Inc. and Miquido
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package elasticsearch
+
+import (
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/miquido/conduit-connector-elasticsearch/destination"
+)
+
+func Specification() sdk.Specification {
+	return sdk.Specification{
+		Name:    "elasticsearch",
+		Summary: "An Elasticsearch destination plugin for Conduit.",
+		Version: "v0.1.0",
+		Author:  "Miquido",
+		DestinationParams: map[string]sdk.Parameter{
+			destination.ConfigKeyVersion: {
+				Default:     "",
+				Required:    true,
+				Description: "The version of the Elasticsearch service.",
```

Comment:
Would be nice to mention what are the possible values (and in which format, e.g. is it 8 or v8). It is mentioned in the ReadMe (which is great!) but these specs will be used to automatically populate options in a UI widget.

Reply:
Gotcha, added 🙂